common/src/main/java/com/wynntils/core/features/properties/StartEnabled.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.core.features.properties;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Used to indicate whether an InternalFeature should begin as enabled or not
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+public @interface StartEnabled {
```

Comment:
Maybe make this `EnabledOnStart`? (Once again, open to better names)

Reply:
Or turn it into `AlwaysEnabled`, but this would imply that it can't be disabled. But my impression is that what we really want with these internal features that are started as enabled. Core functionality.

Reply:
I feel as though we could just have StartDisabled and enable all features by default

Reply:
I think Incomplete's suggestion is the best. Still have no strong opinion on this, tho.

Reply:
Or keep it as it is... We can always fix it later if we need to.